ros/src/foxglove_bridge/src/ros2_foxglove_bridge.cpp

```diff
@@ -1072,6 +1084,24 @@ rclcpp::QoS FoxgloveBridge::determineQoS(const std::string& topic) {
   return qos;
 }
 
+void FoxgloveBridge::onClientConnect() {
+  const auto newCount = ++_clientCount;
+  RCLCPP_DEBUG(this->get_logger(), "Client connected. Total clients: %u", newCount);
```

Comment:
Let's remove these calls to `RCLCPP_DEBUG`, since we have the topic itself as a read of this